roles/common/defaults/main.yml

```diff
@@ -1,5 +1,5 @@
 ---
 # Note: `nave` requires a dot in the version, so for nodejs 4.x, you must
 # specify 4.0 or 4.1 or 4.2 or 4.3 (or what ever minor version in the future)
-node_version: "0.10"
+node_version: "4.5"
```

Comment:
Node 4.5.0 'Argon' includes npm 2.15.9 (per https://github.com/nodejs/node/blob/master/doc/changelogs/CHANGELOG_V4.md#4.5.0)

Diff: https://github.com/npm/npm/compare/v2.14.18...v2.15.9

Not sure if you also want to bump the `npm_version` below or not.